corgi/tasks/management/commands/loadcomposedata.py

```diff
@@ -15,14 +19,30 @@ def add_arguments(self, parser: CommandParser) -> None:
             "compose_names",
             nargs="*",
             type=str,
-            help="Names of composes to load",
+            help="Fetch builds for compose by name",
+        )
+        parser.add_argument(
+            "-s",
+            "--stream",
+            dest="stream",
+            help="Fetch builds for composes in stream",
+        )
+        parser.add_argument(
+            "-i",
```

Comment:
nitpick = we could use -c (for indicating celery) and change default case to without celery